clang/test/ClangScanDeps/modules-context-hash-ignore-macros.c

```diff
@@ -27,6 +27,11 @@
 // CHECK:            "context-hash": "[[HASH_FOO:.*]]"
 // CHECK:            "name": "Mod"
 // CHECK-NEXT:     }
+// CHECK:           "command-line": [
```

Comment:
For my own understanding, can you please explain (in PR comment, no need to update patch) what this test tests and why your PR caused it to change?

Reply:
My understanding for this change is due to the option that was added. An additional command line was added to resulting json file.

Reply:
I see. So "-full-main-file-name" is generated always even in non-DPCPP environment. Is that the intention?

Reply:
That's correct.

Reply:
Why do we need it when we're not doing DPCPP compilation?

Reply:
Good point! Just checked the debug info when compiling with no  option -fsycl without this patch. It is correct! So I think this option can be added on the sycl path.